flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -469,10 +469,10 @@ private static List<TestSetSpec> allTypesBasic() {
                         .fromCase(BIGINT(), DEFAULT_NEGATIVE_BIGINT, 539222987)
                         .fromCase(FLOAT(), DEFAULT_POSITIVE_FLOAT, 123)
                         .fromCase(FLOAT(), DEFAULT_NEGATIVE_FLOAT, -123)
-                        .fromCase(FLOAT(), 9234567891.12, 644633299)
+                        .fromCase(FLOAT(), 9234567891.12, 2147483647)
```

Comment:
All the changes is this file are a consequence of https://issues.apache.org/jira/browse/CALCITE-4861

Reply:
Added release notes to the JIRA issue

Reply:
This line is still not correct. It says:
```
.fromCase(FLOAT(), 9234567891.12, 2147483647)
```

But the argument is not float. It should be:
```
.fromCase(FLOAT(), 9234567891.12f, 2147483647)
```

Reply:
done